pkg/http/api.go

```diff
@@ -409,29 +409,6 @@ func (a *api) onDirectMessage(c *routing.Context) error {
 	return nil
 }
 
-// DEPRECATED
```

Comment:
Removed this function, as this function is deprecated.

Reply:
Yup, I think we can clean up this now. cc/ @yaron2